src/internal/m365/collection/drive/collections_tree_test.go

```diff
@@ -843,73 +844,94 @@ func (suite *CollectionsTreeUnitSuite) TestCollections_PopulateTree_singleDelta(
 				err:            require.NoError,
 				numLiveFiles:   3,
 				numLiveFolders: 4,
-				sizeBytes:      3 * defaultItemSize,
+				sizeBytes:      3 * 42,
```

Comment:
not sure if there's another PR to handle the default (haven't looked ahead yet), but this is technically reverting the code we'd like to have

Reply:
ah, bargle.  rebases.  I'll clean this up again at the end of the bookend.